src/app/navigation/utils.ts

```diff
@@ -128,7 +128,7 @@ export const findPathForParams = (params: object, paths: string[]) => {
   });
 };
 
-export const getQueryForParam = (param: string, value: string, existingQuery?: string | OutputParams) => {
+export const getQueryForParam = (param: string, value: string | string[], existingQuery?: string | OutputParams) => {
```

Comment:
how does this library encode arrays? as i remember it there is no super standard way to do that

Reply:
it duplicates the params, so it looks like `colors=gree&colors=yellow`

Reply:
This seems to be the default behaviour
```js
const search = new URLSearchParams('')
search.append('colors', 'green')
search.append('colors', 'yellow')
search.toString() // "colors=green&colors=yellow"
```

Reply:
There is also `getAll`:
```js
search.getAll('colors') // Array [ "green", "yellow" ]
```